docs/getting-started-apm-server.asciidoc

```diff
@@ -0,0 +1,314 @@
+[[getting-started-apm-server]]
+== Getting Started With APM Server
+
+// This section was copied from the APM Overview
+// Switch to tagged regions when available
+The easiest way to get started with APM Server is by using our
+https://www.elastic.co/cloud/elasticsearch-service[hosted {es} Service] on
+Elastic Cloud. The {es} Service is available on AWS, GCP, and Azure,
+and automatically configures APM Server to work with {es} and {kib}.
+
+[float]
+=== Hosted Elasticsearch Service
```

Comment:
Shouldn't this be _Hosted Elastic Service_?

Reply:
I'll double-check with marketing, but based on https://www.elastic.co/cloud/, I think the actual product name is "Elasticsearch Service"

Reply:
Confirmed we should stick with "Hosted Elasticsearch Service"